### DISCUSSION TITLE
Image crate size since 0.24.8

### DISCUSSION THREAD

**Discussion by nail-garejev-lightcon** [image-rs/image#2205] [Q&A] [answered]
Looking at [image crate size by version](https://crates.io/crates/image/versions), I've noticed that it jumped from ~280 KiB in 0.24.7 to ~8.8 MiB in 0.24.8. 

I've tried searching for, but couldn't find the explanation for such large change. 

Does anybody know the reason? 

![image](https://github.com/image-rs/image/assets/110464833/4f7bf8ab-0ade-4ae5-8857-5c360a3f7a73)

**Comment by 197g** [comment, marked as answer]
Seems that the `Cargo.toml` metadata has changed such that the `include` and `exclude` keys are no longer under the `package` section as they should be. Fix: https://github.com/image-rs/image/pull/2206
